docker/release/dockerfiles/opensearch-dashboards.al2.dockerfile

```diff
@@ -52,12 +52,17 @@ FROM amazonlinux:2
 ARG UID=1000
 ARG GID=1000
 ARG OPENSEARCH_DASHBOARDS_HOME=/usr/share/opensearch-dashboards
+ENV TINI_VERSION=v0.19.0
 
 # Update packages
 # Install the tools we need: tar and gzip to unpack the OpenSearch tarball, and shadow-utils to give us `groupadd` and `useradd`.
 # Install which to allow running of securityadmin.sh
 RUN yum update -y && yum install -y tar gzip shadow-utils which && yum clean all
 
+# Add tini to use as init (PID1) process.
+ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini /bin/tini
+RUN chmod +x /bin/tini
```

Comment:
I kinda prefer to not make it globally available, and add full path in the entrypoint.
Tho I am not strongly against it.

Reply:
probably use 755 specifically instead of +x so it is precise?

Reply:
Sounds good, updated the chmod to 755. Open to path suggestions where we can keep this binary

Reply:
Thanks!

Reply:
As of path, just use `WORKDIR`? It is the home dir of os or osd on docker.
So instead of using `/bin/tini` just use `./tini`?